troupon/static/scss/intlTelInput.scss

```diff
@@ -0,0 +1,281 @@
+$arrowPadding: 4px !default;
+$arrowColor: #555 !default;
+
+$inputPadding: 6px !default;
+$selectedFlagWidth: $flagWidth + $flagPadding + $arrowWidth + (2 * $arrowPadding) !default;
+
+// image related variables
+$flagsImagePath: "../img/" !default;
+$flagsImageName: "flags" !default;
+$flagsImageExtension: "png" !default;
+
+.intl-tel-input {
+  // need position on the container so the selected flag can be
+  // absolutely positioned over the input
+  position: relative;
+  // keep the input's default inline properties
+  display: inline-block;
+
+  // paul irish says this is ok
+  // http://www.paulirish.com/2012/box-sizing-border-box-ftw/
+  * {
+    box-sizing: border-box;
+    -moz-box-sizing: border-box;
+  }
+
+  .hide {
+    display: none;
+  }
+  // need this during init, to get the height of the dropdown
+  .v-hide {
+    visibility: hidden;
+  }
+
+  // specify types to increase specificity e.g. to override bootstrap v2.3
+  input, input[type=text], input[type=tel] {
+    position: relative;
+    // input is bottom level, below selected flag and dropdown
+    z-index: 0;
+
+    // any vertical margin the user has on their inputs would no longer work as expected
+    // because we wrap everything in a container div. i justify the use of !important
+    // here because i don't think the user should ever have vertical margin here - when
+    // the input is wrapped in a container, vertical margin messes up alignment with other
+    // inline elements (e.g. an adjacent button) in firefox, and probably other browsers.
+    margin-top: 0 !important;
+    margin-bottom: 0 !important;
+
+    // make space for the selected flag
+    // Note: no !important here, as the user may want to tweak this so that the
+    // perceived input padding matches their existing styles
+    padding-left: $selectedFlagWidth + $inputPadding;
+
+    // any margin-left here will push the selected-flag away
+    margin-left: 0;
+
+    // when user presses invalid key, we add a class which instantly adds a bg color,
+    // then when we remove that class, this does a quick fade back to no bg
+    transition: background-color 100ms ease-out;
+  }
+  // when user presses invalid key, add this class temporarily to change bg color
+  input.iti-invalid-key {
+    // override the default transition state to show the change instantly
+    transition: background-color 0s;
+    background-color: #FFC7C7;
+  }
+
+  .flag-container {
+    // positioned over the top of the input
+    position: absolute;
+    // full height
+    top: 0;
+    bottom: 0;
+    // prevent the highlighted child from overlapping the input border
+    padding: $borderWidth;
+  }
+  // hover state - show flag is clickable
+  .flag-container:hover {
+    cursor: pointer;
+    .selected-flag {
+      background-color: $hoverColor;
+    }
+  }
+  // disable hover state when input is disabled
+  input[disabled] + .flag-container:hover, input[readonly] + .flag-container:hover {
```

Comment:
Avoid qualifying attribute selectors with an element.<br>Each selector in a comma sequence should be on its own single line